File: 3872-find-most-frequent-vowel-and-consonant/find-most-frequent-vowel-and-consonant.java
class Solution {
    public int maxFreqSum(String s) {
        int[] fr = new int[26];
        for(char i:s.toCharArray()){
            fr[i-'a']++;
        }
        int maxv = 0,maxc = 0;
        for(int i=0;i<fr.length;i++){
            if(i==0 || i==4 || i==8 || i==14 || i==20){
                if(maxv<fr[i]) maxv = fr[i];
            }else{
                if(maxc<fr[i]) maxc = fr[i];
            }
        }
        return maxc+maxv;
    }
}

File: 3872-find-most-frequent-vowel-and-consonant/find-most-frequent-vowel-and-consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        freq = {}
        for i in s:
            freq[i] = freq.get(i,0)+1
        
        cmax = 0
        vmax = 0
        vowels = "aeiou"
        for key,value in freq.items():
            if key in vowels:
                vmax = max(vmax,value)
            else:
                cmax = max(cmax,value)
        
        return vmax+cmax
